test/controllers/user.test.js

```diff
@@ -127,3 +127,26 @@ describe('usersController.users', () => {
         })
     ));
 });
+
+describe('usersController.buy', () => {
+  it('Buys one album', async () => {
+    const token = await createAndSignInUser();
+    const response = await request.post('/albums/4').set('token', token);
+
+    const ua = await models.userAlbums.findOne({ where: { albumId: response.body.album.id } });
+    expect(ua.dataValues.albumId).toBe(response.body.album.id);
+  });
+
+  it('Tries to buy the same album and fails', async () => {
+    const token = await createAndSignInUser();
+    await request.post('/albums/4').set('token', token);
+    const response = await request.post('/albums/4').set('token', token);
+
+    expect(response.body.internal_code).toBe('album_already_purchased');
+  });
+
+  it('Tries to buy an album without login and fails', async () => {
+    const response = await request.post('/albums/4').set('token', 'no-valid-token');
+    expect(response.body.internal_code).toBe('invalid_token');
+  });
```

Comment:
I don't see you mocking the external service in any of the tests.  That's no bueno, since you end up depending on the service to be up for the test to pass.